src/port/switch/SwitchImpl.cpp

```diff
@@ -74,6 +76,51 @@ void Ship::Switch::SetupFont(ImFontAtlas* fonts) {
     plExit();
 }
 
+void Ship::Switch::ImGuiSwapABXY(int startEvent) {
+    if (CVarGetInteger("gOpenMenuBar", 0)) {
+        return;
+    }
+
+    auto& input_queue = ImGui::GetCurrentContext()->InputEventsQueue;
+    for (int n = start_event; n < input_queue.Size; n++) {
```

Comment:
```suggestion
    for (int n = startEvent; n < input_queue.Size; n++) {
```

to fix a build error